### PR TITLE
Feature/fix recurrence token

### PR DESCRIPTION
**Pull request by SteffenSH**
Fixed problems related to recurrring credit card payments.

### CODE REVIEW

**Review by NullableInt — APPROVED** [review]

File: src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentResponse.cs [hunk 1]
```diff
@@ -99,6 +101,7 @@ public class PaymentResponseObject
         public string PayerReference { get; }
         public string InitiatingSystemUserAgent { get; }
         public PricesListResponse Prices { get; }
+        public Guid? RecurrenceToken { get; }
```
Comment (NullableInt) [inline src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentResponse.cs:104]: Shouldn't this be a `string`?
https://developer.swedbankpay.com/payment-instruments/card/other-features#recur It's defined as a string here.
Any opinions @asbjornu ?

Comment (asbjornu) [reply src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentResponse.cs:104]: Agreed, the format of tokens shouldn't be restricted or interpreted in any way. They are just opaque identifiers.

Comment (SteffenSH) [reply src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentResponse.cs]: Ok. I understand that from your perspective you want to keep the options open.
I set it to Guid to make it more predictable on the client side, but I have now changed it to string (of unlimited length).

**Review by NullableInt — APPROVED** [review]

**Comment by asbjornu** [comment]
@SteffenSH, can you please try the latest `2.0.0` beta to see whether this is now fixed there?

**Comment by asbjornu** [comment]
I think this should be covered by the latest 2.0.0 release, so closing. Please reopen a new (rebased) PR if there's still stuff to do here.

**Pull request closed without merging**